### PR TITLE
Move CI to GHA

### PR DESCRIPTION
**Pull request by w13915984028**
issue: https://github.com/harvester/harvester/issues/5327

### CODE REVIEW

File: scripts/build [hunk 1]
```diff
@@ -11,7 +11,13 @@ if [ "$(uname)" = "Linux" ]; then
...
-    GOOS=darwin go build -ldflags "$LINKFLAGS" -o bin/eventrouter-darwin
-    GOOS=windows go build -ldflags "$LINKFLAGS" -o bin/eventrouter-windows
-fi
+
+
+for arch in "amd64" "arm64"; do
+    GOARCH="$arch" CGO_ENABLED=0 go build -ldflags "$LINKFLAGS $OTHER_LINKFLAGS" -o bin/eventrouter-"$arch"
+done
+
+#if [ "$CROSS" = "true" ] && [ "$ARCH" = "amd64" ]; then
```
Comment (bk201) [inline scripts/build:20]: Suggest removing unused codes.

File: scripts/package [hunk 2]
```diff
@@ -5,14 +5,12 @@ source $(dirname $0)/version
...
-cp bin/eventrouter dist/artifacts/eventrouter${SUFFIX}
-
 IMAGE=${REPO}/eventrouter:${TAG}
 DOCKERFILE=package/Dockerfile
 if [ -e ${DOCKERFILE}.${ARCH} ]; then
     DOCKERFILE=${DOCKERFILE}.${ARCH}
 fi
 
-docker build -f ${DOCKERFILE} -t ${IMAGE} .
+#docker build -f ${DOCKERFILE} -t ${IMAGE} .
```
Comment (bk201) [inline scripts/package:14]: Suggest removing unused codes.

**Review by bk201 — COMMENTED** [review]
lgtm, but need an EIO ticket to migrate secrets.

**Review by Vicente-Cheng — APPROVED** [review]
lgtm, thanks!

**Review by bk201 — APPROVED** [review]

**Pull request merged**